server/queue.go

```diff
@@ -605,11 +606,33 @@ func (s *Server) clientMadeIRODSConnections(c *gin.Context) {
 	defer s.mapMu.Unlock()
 
 	s.iRODSConnections[hostPID] += n
-	s.sendSlackMessage(slack.Info, strconv.Itoa(s.totalIRODSConnections())+" iRODS connections open")
+	s.createAndSendIRODSSlackMsg()
 
 	c.Status(http.StatusOK)
 }
 
+func (s *Server) createAndSendIRODSSlackMsg() {
```

Comment:
This seems very similar (identical?) to uploadTracker functionality, so shouldn't be reimplemented here. It makes sense to break out the functionality from uploadTracker to a slackMessageDebouncer and use it in uploadTracker and a new connectionTracker.